lib/puppet/util/windows/taskscheduler2_v1task.rb

```diff
@@ -0,0 +1,559 @@
+require 'puppet/util/windows/taskscheduler2'
+require 'puppet/util/windows/taskscheduler' # Needed for the WIN32::ScheduledTask flag constants
+
+# This class is used to manage V1 compatible tasks using the Task Scheduler V2 API
+# It is designed to be a binary compatible API to puppet/util/windows/taskscheduler.rb but
+# will only surface the features used by the Puppet scheduledtask provider
+#
+class Puppet::Util::Windows::TaskScheduler2V1Task
+  # The error class raised if any task scheduler specific calls fail.
+  class Error < Puppet::Util::Windows::Error; end
```

Comment:
Defined here as well?

Reply:
Spec tests were failing as it was expecting these definitions.